### PR TITLE
Add APENFT token(BEP20 and ERC20)

### PR DESCRIPTION
**Pull request by Benson0224**
As introduced on the APENFT homepage(https://fundation.apenft.io/)  APENFT was deployed to several different chains. 

This PR is going to add the logo info for Etherem and BSC contract. 

APENFT Contract on Tron: TFczxzPhnThNSqr5by8tvxsdCFRRz6cPNq
APENFT Contract on Ethereum: 0x198d14f2ad9ce69e76ea330b374de4957c3f850a
APENFT Contract on Binance Smart Chain: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

### CODE REVIEW

**Comment by Cryptocool1** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .

**Comment by Benson0224** [comment]
APENFT is originally launched on TRON network. 

it has more than 2 million holders and more than 5 millions transfer.  you can get the general info from the attached screenshot and you can also check the detail at https://tronscan.org/#/token20/TFczxzPhnThNSqr5by8tvxsdCFRRz6cPNq/holders 

the BEP20 APENFT and ERC20 APENFT are newly bridged into BSC and Etherem,  it was also newly announced on TRON Dao twitter https://twitter.com/trondao/status/1510870344026177536  and it is also vibile on APENFT homepage https://fundation.apenft.io/ 

Please help to re-open this PR so we can move on. thanks

image
image

image

**Comment by vcoolish** [comment]
we don't support TRC721

**Comment by vcoolish** [comment]
If your token is TRC20 feel free to add it

**Comment by Benson0224** [comment]
@Cryptocool1  please help to re-open this PR, thanks.

File: blockchains/ethereum/assets/0x198d14F2Ad9CE69E76ea330B374DE4957C3F850a/info.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "APENFT",
+    "symbol": "NFT",
+    "type": "ERC20",
+    "decimals": 6,
+    "description": "APENFT Fund was born with the mission to register worid-class artworks as NFTs on blockchain and aim to be the ARK Funds in the NFT space to buld a bridge between top-notch artists and blockchain, and to support the growth of native crypto NFT artists.",
```
Comment (bogvas194-wq) [inline blockchains/ethereum/assets/0x198d14F2Ad9CE69E76ea330B374DE4957C3F850a/info.json]: TXzJrZd16dbQcN9zU3RYVyVrco9gb1FFLc

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "APENFT",
+    "symbol": "NFT",
+    "type": "BEP20",
+    "decimals": 6,
+    "description": "APENFT Fund was born with the mission to register worid-class artworks as NFTs on blockchain and aim to be the ARK Funds in the NFT space to buld a bridge between top-notch artists and blockchain, and to support the growth of native crypto NFT artists.",
+    "website": "https://apenft.io/",
+    "explorer": "https://bscscan.com/token/0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca",
+    "status": "active",
+    "id": "0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa",
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "APENFT",
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: TXzJrZd16dbQcN9zU3RYVyVrco9gb1FFLc

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            "name": "twitter",
+            "url": "https://twitter.com/apenftorg"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/apenftEN"
+        },
+        {
+            "name": "whitepaper",
+            "url": "https://fundation.apenft.io/book/APENFT%20White%20Paper.pdf"
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: BEP20 - 0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        },
+        {
+            "name": "medium",
+            "url": "https://apenftorg.medium.com/"
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "APENFT",
+    "symbol": "NFT",
+    "type": "BEP20",
+    "decimals": 6,
+    "description": "APENFT Fund was born with the mission to register worid-class artworks as NFTs on blockchain and aim to be the ARK Funds in the NFT space to buld a bridge between top-notch artists and blockchain, and to support the growth of native crypto NFT artists.",
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    "website": "https://apenft.io/",
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    "explorer": "https://bscscan.com/token/0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca",
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20ee7b720f4e4c4ffcb00c4065cdae55271aecca

File: blockchains/ethereum/assets/0x198d14F2Ad9CE69E76ea330B374DE4957C3F850a/info.json [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    "website": "https://apenft.io/",
+    "explorer": "https://etherscan.io/token/0x198d14f2ad9ce69e76ea330b374de4957c3f850a",
```
Comment (bogvas194-wq) [inline blockchains/ethereum/assets/0x198d14F2Ad9CE69E76ea330B374DE4957C3F850a/info.json]: 0x198d14f2ad9ce69e76ea330b374de4957c3f850a

File: blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+    "id": "0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa",
+    "links": [
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/apenftorg"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/apenftEN"
+        },
```
Comment (bogvas194-wq) [inline blockchains/smartchain/assets/0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa/info.json]: 0x20eE7B720f4E4c4FFcB00C4065cdae55271aECCa

**Pull request merged**